skimage/segmentation/_clear_border.py

```diff
@@ -16,7 +16,9 @@ def clear_border(labels, buffer_size=0, bgval=0, in_place=False):
         Cleared objects are set to this value.
     in_place : bool, optional
         Whether or not to manipulate the labels array in-place.
-
+    mask : ndarray of bool, same shape as `image`, optional.
+        Image data mask. Objects in labels image overlapping with
+        True pixels of mask will be removed.
```

Comment:
Two points:

- I think we should reverse this logic. In the rest of the library, `mask==True` corresponds to the bits of the image that are kept/processed. So for consistency I think the same should be true here.
- Add a note that the borders are ignored if `mask` is given.

Reply:
ok, I think that it is logical to keep it consitent. I'll commit the corresponding changes.

Reply:
Hi @PeterJackNaylor, thank you for working on that!
I would remove the info "same shape as `image`" here and put it in the description. I'd put also `labels` in its place, since `image` is not one of the parameters, and not everyone will check the code :wink: Something like that:

```suggestion
    mask : (M[, N[, ..., P]]) array of bool, optional.
        Image data mask with same shape as `labels`. Objects in labels
        image overlapping with True pixels of mask will be removed.
```

What do you both (you and @jni) think about that?
Thanks again!

Reply:
@alexandrejaguar good suggestion but:

Objects in labels image overlapping with False pixels of mask will be removed.

;)